.readthedocs.yaml

```diff
@@ -8,18 +8,26 @@ version: 2
 build:
   os: "ubuntu-22.04"
   tools:
+      # This python version needs to be compatible with SCT, see setup.py
       python: "3.9"
 
-# Build documentation in the docs/ directory with Sphinx
-sphinx:
-  configuration: documentation/source/conf.py
-  fail_on_warning: true
-
 python:
   install:
     - requirements: requirements.txt
-    - method: pip
-      path: .
+    # Install SCT itself
+    - path: .
```

Comment:
Good catch! It does indeed appear that [`pip` is the default option](https://docs.readthedocs.io/en/stable/config-file/v2.html#packages) for `method:`.